api/v1/cluster_types.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ClusterSpec defines the desired state of Cluster
+type ClusterSpec struct {
+	// The git location of the cluster template
+	ClusterTemplate RepoSpec `json:"clusterTemplate"`
+	// Optional override specification
+	Override *OverrideSpec `json:"override,omitempty"`
+	// Optional autoscaler specification
+	Autoscaler *AutoscalerSpec `json:"autoscaler,omitempty"`
+	// Optional Arlon Helm chart specification if defaults are not desired
+	ArlonHelmChart *RepoSpec `json:"arlonHelmChart,omitempty"`
```

Comment:
This refers to the manifests in `pkg/cluster/manifest` correct?

Reply:
Yes